Readme.md

```diff
@@ -271,26 +318,45 @@ topics_list = [t for t in topics if not (
     (t.standalone_title == "California Standards Test: Algebra I" and t.id != "algebra-i") or
     (t.standalone_title == "California Standards Test: Geometry" and t.id != "geometry-2"))
     ]
+
 # GOOD
-bad_title_and_ids = [("California Standards Test: Algebra I", "algebra-i"),
-                     ("California Standards Test: Geometry", "geometry-2"),
-                    ]
-topics_list = [t for t in topics
-               if not (t.standalone_title, t.id) in bad_title_and_ids]
+bad_title_and_ids = [
+    ("California Standards Test: Algebra I", "algebra-i"),
+    ("California Standards Test: Geometry", "geometry-2"),
+]
+topics_list = [
+    t for t in topics
+    if not (t.standalone_title, t.id) in bad_title_and_ids
+]
 ```
 
-## Indenting continued lines properly
-
-When a logical statement is split into multiple lines and is followed by an indented line, the continued lines should be indented further to set them apart from the next logical statement.
-
-In the following example taken from http://www.python.org/dev/peps/pep-0008/#maximum-line-length, notice how the second and third lines of the if condition are indented further to differentiate them from the actual raise statement within the if block:
+Class initializers and if statements:
 
 ```py
+# BAD
 class Rectangle(Blob):
-    def __init__(self, width, height,
-                     color='black', emphasis=None, highlight=0):
+    def __init__(self, width,
+        height, color='black',
+        emphasis=None, highlight=0):
         if (width == 0 and height == 0 and
                 color == 'red' and emphasis == 'strong' or
                 highlight > 100):
             raise ValueError("sorry, you lose")
+
+# GOOD
+class Rectangle(Blob):
+    def __init__(
+        self,
+        width,
+        height,
+        color='black',
+        emphasis=None,
+        highlight=0
+    ):
```

Comment:
this looks funny to the uninitiated, but when you think about it, this is highly legible, and the block boundaries are very clear.

overall, love this style.